src/Files.App/Services/Settings/GeneralSettingsService.cs

```diff
@@ -267,6 +267,12 @@ public string UserId
 			set => Set(value);
 		}
 
+		public ShellPaneAlignmentDirection ShellPaneAlignmentDirection
```

Comment:
Wouldn't this force all tabs to have the same orientation?

Reply:
True, it overrides.
I think we should include this in `LastOpnedTabsList`?

Reply:
As an arg?

Reply:
Yeah probably adding the enum to the parameter class. I’ll try.

Reply:
Once we implement #14875, we'll be able to add properties for each tab in the list. In the meantime, we can just add it as a param.

Reply:
Done.